devices/roome.js

```diff
@@ -0,0 +1,15 @@
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const e = exposes.presets;
+
+module.exports = [
+    {
+        zigbeeModel: ['MULTI-MECI--EA01'],
+        model: 'HSC1-WD-0',
```

Comment:
Can you provide an URL of this device? (with picture)

Reply:
Only find the review link
https://www.smarthome.com.cn/xinpin/796.html
The manufacturer may be
http://en.mirsz.cn/detail-55.shtml